A <%= @exception.class %> occurred <%= @exception_source %>:

  <%=raw @exception.message %>
  <%=raw @backtrace.first %>

<%=raw @sections.map { |section| render_section(section) }.join %>
